<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Baza</class>
 <widget class="QDialog" name="Baza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Сортировки</string>
     </property>
     <widget class="QRadioButton" name="HeapSort">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Пирамидальная</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="QuickSort">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Быстрая</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadixSort">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Пузырька</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ShakeSort">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Шейкерная</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сортировать</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Сортировка по годам</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>341</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Время сортировки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="vrem_god"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Количество сравнений:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sravn_god"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Количество перестановок:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pers_god"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Сортировка по населению</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>341</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Время сортировки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="vrem_nas"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Количество сравнений:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sravn_nas"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Количество перестановок:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pers_nas"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
